src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return cost - 50;
+  }
+
+  if (days >= 3) {
+    return cost - 20;
+  }
```

Comment:
you're done logic-wise 👍 
you can also improve the code readability by telling what numbers 20 and 50 are:
```
const discountLower = 20;
const discountHigher = 20;
```